Jellyfin.Plugin.OpenSubtitles/OpenSubtitleDownloader.cs

```diff
@@ -207,124 +168,143 @@ await Task.Delay(250, cancellationToken)
         {
             if (string.IsNullOrWhiteSpace(id))
             {
-                throw new ArgumentNullException(nameof(id));
+                throw new ArgumentException("Missing param", nameof(id));
             }
 
+            if (_login?.User?.RemainingDownloads <= 0)
+            {
+                if (_limitReset < DateTime.UtcNow)
+                {
+                    _logger.LogDebug("Reset time passed, forcing a new login");
+                    // force login because the limit resets at midnight
```

Comment:
Is a re-login necessary?

Reply:
It is not, I changed it so it only updates the User part of the current login/session (fccde9883c49f7d46dd459a8ce712f512751c378)